src/ploigos_step_runner/step_implementers/container_image_static_vulnerability_scan/openscap.py

```diff
@@ -33,6 +33,9 @@
                                                        remote resources and this is not True. \
                                                        For disconnected environments the remote \
                                                        internal mirror.
+'oscap-severity'               | No        | ''      | Severity threshold for failing a step. \
```

Comment:
can you add the expected/allowed values here

Reply:
I'll get that in tomorrow.